src/applications/disability-calculator/utils/helpers.js

```diff
@@ -1,57 +1,31 @@
-// helper function to round to nearest 10
-function roundRating(num) {
-  const ratingWithDecimal = parseFloat(num).toFixed(0.1);
-  return Math.round(ratingWithDecimal / 10) * 10;
+export function getRatings(disabilities) {
+  return disabilities.map(disability => disability.rating);
 }
 
-export function pullRatingsFromState(arr) {
-  const allRatings = [];
-  arr.forEach(e => {
-    allRatings.push(e.rating);
-  });
-  return allRatings;
+export function canCalculate(ratings) {
+  return ratings.filter(r => !!r).length >= 2;
 }
 
-export function calculateRating(arr) {
-  const ratingArr = pullRatingsFromState(arr);
-  const sortedArr = ratingArr.sort((a, b) => b - a);
-  let a;
-  let b;
-  let x;
+export function calculateCombinedRating(ratings) {
+  const ratingsSortedDesc = ratings.sort((a, b) => b - a);
 
-  while (sortedArr.length > 1) {
-    a = 100 - sortedArr[0];
-    b = (sortedArr[1] * a) / 100;
-    x = sortedArr[0] + b;
-    sortedArr.splice(0, 2, x);
-  }
+  const combinedRating = ratingsSortedDesc.reduce(
+    (currentCombinedRating, nextRating) => {
+      if (!currentCombinedRating) return nextRating;
 
-  if (sortedArr.length === 1) {
-    const lastCalcualtedRating = sortedArr[0];
-    let result = roundRating(lastCalcualtedRating);
+      const a = 100 - currentCombinedRating;
+      const b = nextRating * (a / 100);
+      const nextCombinedRating = currentCombinedRating + b;
 
-    const actualRating = Number(parseFloat(lastCalcualtedRating).toFixed(2));
-    if (result > 100) {
-      result = 100;
-    }
-    return [result, actualRating];
-  }
-  return [undefined, undefined];
-}
-// calculateRating will return array with two elements. first element in array is rounded rating
-// and second element is the actual rating
+      return Math.round(nextCombinedRating);
+    },
+    null,
+  );
+
+  const combinedRatingRounded = Math.round(combinedRating / 10) * 10;
 
-// this is a validator to check if there are at least two ratings
-export function shouldCalculate(ratingsArr) {
-  const checkIfTwo = ratingsArr;
-  const returnArr = [];
-  checkIfTwo.forEach(e => {
-    if (typeof e === 'number') {
-      returnArr.push(e);
-    }
-  });
-  if (returnArr.length >= 2) {
-    return true;
-  }
-  return false;
+  return {
+    exact: combinedRating,
+    rounded: combinedRatingRounded,
+  };
```

Comment:
`calculateCombinedRating` is essentially converting the pattern of using the data tables on this following page into a function - https://www.benefits.va.gov/compensation/rates-index.asp.

There's some [whiteboard](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18214) work in the epic showing what the formula looks like.